scripts/setup

```diff
@@ -28,9 +28,9 @@ fi
 if ! which verilator >/dev/null; then
     missing+=(verilator libevent-dev libjson-c-dev)
 fi
-if ! (apt list -i | grep libusb-1.0-0-dev) ; then
-    missing+=(libusb-1.0-0-dev libftdi1-dev)
-fi
+#if ! (apt list -i | grep libusb-1.0-0-dev) ; then
```

Comment:
Oh, I missed this change before. This breaks the Github actions, we need libusb headers for compiling flashrom.

Reply:
Right, I've also noticed that I've introduced a change in `common/Makefile`, which was meant to be only temporary. Thanks for spotting it, it should be fixed now.